src/main/resources/webroot/management/openapi.yml

```diff
@@ -23,14 +23,18 @@ paths:
                 password:
                   description: Password for the new user.
                   type: string
+                role:
+                  description: Role for the new user. Role "mygroup_manager" allows to access data of users with role "mygroup_user".
```

Comment:
Since multiple roles are possible per user, I do ask myself, why we need to use this underscore stuff? Couldn't we simply put users into two or more groups, like "mygroup, user" and "mygroup, user, manager"?

Reply:
1) "someGroup_manager" sounds more explicit than "someGroup", "manager" to me.

The first thing tells you, that this user is a manager of the group.
The second just says that the user is a manager (of what?) and part of the group.

2) we would need to ensure all "managers" have a "group" and all "group" parties have a assignment like "user", "manager", or so. (Or don't we?)

3) I don't really see how this would make things easier right now, but I also don't know what kind of user management we'll have in the future. If we want to see all users of a group, no matter the assignment, these more complex roles might make sense as it might be faster to search for "group" than "group_user" and "group_manager" (we would need to know which kind of assignments exist, also, for the second.

4) Would we need to keep a reference about what type of role one such is? E.g. a "group" or a "assignment"? E.g. with underscore (epic_group, lazy_group, user, manager) or with a roles collection (with a field "type") or we put the knowledge about which assignments there are (user, manager, admin) hard-coded into the code. We might want to add some asserts to the user creation handler then, that there is no "admin, manager" or "user, manager", oder so by accident?

=> I don't really have a strong opinion on this. My code suggestions was just the most simple way I could think of without much code to write.

Reply:
=> ok was just a misunderstanding - the underscored roles are *not* parsed into multiple roles. Thus, we can keep if simple like this.